client-bagel-buddy/src/index.css

```diff
@@ -20,6 +20,7 @@ code {
   --primary: #BF7217;
   --gray: #717171;
   --white: #FFF;
+  --popup:  #AFAFAF;
```

Comment:
don't name it `popup` hahahaha No one knows what that color is, consider lightGray, darkGray.... something more intuitive